scripts/regressor_finder.py

```diff
@@ -120,9 +120,10 @@ def init_mapping(self):
         logger.info("Downloading Mercurial <-> git mapping file...")
```

Comment:
Please rebase on top of master, as the changes in this and the following file should not be here